src/sentry/static/sentry/app/views/alerts/rules/index.tsx

```diff
@@ -0,0 +1,227 @@
+import {RouteComponentProps} from 'react-router/lib/Router';
+import DocumentTitle from 'react-document-title';
+import React from 'react';
+import styled from '@emotion/styled';
+import flatten from 'lodash/flatten';
+
+import {t, tct} from 'app/locale';
+import {IconCheckmark, IconArrow} from 'app/icons';
+import {Organization, Project} from 'app/types';
+import {IssueAlertRule} from 'app/types/alerts';
+import {PageContent} from 'app/styles/organization';
+import {trackAnalyticsEvent} from 'app/utils/analytics';
+import {Panel, PanelBody, PanelHeader} from 'app/components/panels';
+import AsyncComponent from 'app/components/asyncComponent';
+import EmptyMessage from 'app/views/settings/components/emptyMessage';
+import ExternalLink from 'app/components/links/externalLink';
+import Link from 'app/components/links/link';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import Pagination from 'app/components/pagination';
+import Projects from 'app/utils/projects';
+import withOrganization from 'app/utils/withOrganization';
+import GlobalSelectionHeader from 'app/components/organizations/globalSelectionHeader';
+import {addErrorMessage} from 'app/actionCreators/indicator';
+
+import AlertHeader from '../list/header';
+import {isIssueAlert} from '../utils';
+import {TableLayout} from './styles';
+import RuleListRow from './row';
+
+const DEFAULT_SORT: {asc: boolean; field: 'date_added'} = {
+  asc: false,
+  field: 'date_added',
+};
+const DOCS_URL =
+  'https://docs.sentry.io/workflow/alerts-notifications/alerts/?_ga=2.21848383.580096147.1592364314-1444595810.1582160976';
+
+type Props = RouteComponentProps<{orgId: string}, {}> & {
+  organization: Organization;
+};
+
+type State = {
+  ruleList?: IssueAlertRule[];
+};
+
+class AlertRulesList extends AsyncComponent<Props, State & AsyncComponent['state']> {
+  getEndpoints(): [string, string, any][] {
+    const {params, location} = this.props;
+    const {query} = location;
+
+    return [
+      [
+        'ruleList',
+        `/organizations/${params && params.orgId}/combined-rules/`,
+        {
+          query,
+        },
+      ],
+    ];
+  }
+
+  tryRenderEmpty() {
+    const {ruleList} = this.state;
+    if (ruleList && ruleList.length > 0) {
+      return null;
+    }
+
+    return (
+      <EmptyMessage
+        size="medium"
+        icon={<IconCheckmark isCircled size="48" />}
+        title={t('No alert rules exist for these projects.')}
+        description={tct('Learn more about [link:Alerts]', {
+          link: <ExternalLink href={DOCS_URL} />,
+        })}
+      />
+    );
+  }
+
+  handleDeleteRule = async (projectId: string, ruleId: string) => {
+    const {params} = this.props;
+    const {orgId} = params;
+
+    try {
+      await this.api.requestPromise(
+        `/projects/${orgId}/${projectId}/alert-rules/${ruleId}/`,
```

Comment:
Was just testing this out and you may need to provide a different endpoint depending on deleting a metric alert vs issue alert.

Deleting an issue alert would be at
`DELETE: /projects/${orgId}/${projectId}/rules/${ruleId}/`

Reply:
nice catch! thanks for testing